examples/minimalMilkdrop/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8" />
+</head>
+
+<body>
+    <div id='app'>
+        <!-- Webamp will attempt to center itself within this div -->
+    </div>
+    <script src="https://unpkg.com/webamp@1.3.1/built/webamp.bundle.min.js"></script>
+    <script src="https://unpkg.com/butterchurn@2.6.7/lib/butterchurn.min.js"></script>
+    <script src="https://unpkg.com/butterchurn-presets@2.4.7/lib/butterchurnPresets.min.js"></script>
+    <script>
+        const Webamp = window.Webamp;
+        new Webamp({
+            initialTracks: [{
+                metaData: {
+                    artist: "DJ Mike Llama",
+                    title: "Llama Whippin' Intro"
+                },
+                url: "https://cdn.jsdelivr.net/gh/captbaritone/webamp@43434d82cfe0e37286dbbe0666072dc3190a83bc/mp3/llama-2.91.mp3",
+                duration: 5.322286
+            }],
+            __butterchurnOptions: {
+                importButterchurn: () => Promise.resolve(butterchurn),
+                getPresets: () => {
+                    const presets = butterchurnPresets.getPresets();
+                    return Object.keys(presets).map((name) => {
+                        return {
+                            name,
+                            butterchurnPresetObject: presets[name]
+                        };
+                    });
+                },
+                butterchurnOpen: true
+            },
+            __initialWindowLayout: {
+                MAIN: { position: { x: 0, y: 0 } },
+                EQUALIZER: { position: { x: 0, y: 116 } },
+                PLAYLIST: { position: { x: 0, y: 232 }, size: [0, 4] },
+                MILKDROP: { position: { x: 275, y: 0 }, size: [7, 12] }
```

Comment:
figured it out. These keys should be lower case.

Reminds me that we need to pick a public API for this. Any ideas? Is this, more or less, right?

Reply:
hahahah, yup! That was it :-X

yea, not in love with this, but not sure how to make it more cleaner. WIll give it some thought